ocs_ci/utility/utils.py

```diff
@@ -1541,7 +1541,10 @@ def ceph_health_check(namespace=None, tries=20, delay=30):
 
     """
     return retry(
-        (CephHealthException, CommandFailed), tries=tries, delay=delay, backoff=1
+        (CephHealthException, CommandFailed, TimeoutExpiredError),
```

Comment:
From the problematic job I see the exception was from subprocess and not ours.
```
        if _time() > endtime:
>           raise TimeoutExpired(self.args, orig_timeout)
E           subprocess.TimeoutExpired: Command '['oc', '-n', 'openshift-storage', 'exec', 'rook-ceph-tools-7955584fbd-xd8cn', '--', 'ceph', 'health']' timed out after 600 seconds

/usr/lib64/python3.6/subprocess.py:891: TimeoutExpired```
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/14031/consoleFull

So you will need to import TimeoutExpired from subprocess package.

You will need to import like:

```python
from subprocess import TimeoutExpired
```
```suggestion
        (CephHealthException, CommandFailed, TimeoutExpired),
```
And here to change to above.